package.json

```diff
@@ -18,6 +18,7 @@
     "match-sorter": "^6.3.1",
     "react": "^18.2.0",
     "react-axios": "^2.0.6",
+    "react-burger-menu": "^3.0.9",
```

Comment:
Please create custom burger menu. Not dependency from npm

Reply:
Deleted react-burger-menu: 3.0.9 and create my custom component